### PR TITLE
Update referral letter templates

### PR DESCRIPTION
**Pull request by kewitham**
[Issue 1452](https://github.com/usdoj-crt/crt-portal-management/issues/1452)

## What does this change?
When the DRS referral letters were added, the from line and re line was hard coded where only DRS could use those letters. Other sections need to send some reports to those agencies as well so this PR replaces the hard coded text with the section_name markdown so it is populated according to the section selected.

## Screenshots (for front-end PR):
image

### Author

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [x] Check for, document, and establish a testing plan for any behavior that may vary across environments or is otherwise difficult to test.
+ [ ] Check for [accessibility](https://github.com/usdoj-crt/crt-portal/blob/develop/docs/a11y_plan.md).
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [ ] Check for any behavior that may vary across environments or is difficult to test, and ensure that it is well-understood, documented, and that there is a testing plan in place.
+ [ ] Re-check for [accessibility](https://github.com/usdoj-crt/crt-portal/blob/develop/docs/a11y_plan.md).
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

File: crt_portal/cts_forms/migrations/0161_auto_20230120_1433.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+            model_name='report',
+            name='crt_reciept_year',
+            field=models.PositiveIntegerField(blank=True, null=True, verbose_name=django.core.validators.MaxValueValidator(2023)),
+        ),
+        migrations.AlterField(
+            model_name='report',
+            name='last_incident_year',
+            field=models.PositiveIntegerField(blank=True, null=True, verbose_name=django.core.validators.MaxValueValidator(2023, message='Date can not be in the future.')),
+        ),
+    ]
```
Comment (okkays) [inline crt_portal/cts_forms/migrations/0161_auto_20230120_1433.py:24]: this is odd, what caused this to be included?

**Review by okkays — CHANGES_REQUESTED** [review]
Nice - one question about the included migration

Also just curious, what was the issue with the duplicate select items?

File: crt_portal/cts_forms/migrations/0161_auto_20230120_1433.py (hunk 1 above)
Comment (kewitham) [reply crt_portal/cts_forms/migrations/0161_auto_20230120_1433.py]: Removed! I think I inadvertently added this when I was trying to get the templates to show up properly

**Comment by kewitham** [comment]
> Nice - one question about the included migration
> 
> Also just curious, what was the issue with the duplicate select items?

@okkays when I first updated the select options it was only updating and creating new options without deleting the others so I had to run a script to remove the other options

**Comment by okkays** [comment]
@kewitham Based on that doc, looks like this command will clean them up:

> If you want to manually run the command to add or update response templates without restarting the server, run this command:
> 
> ```sh
> docker-compose run web python /code/crt_portal/manage.py update_response_templates
> ```

which appears to be handled in our circleci workflow: https://github.com/usdoj-crt/crt-portal/blob/develop/.circleci/config.yml

**Review by okkays — APPROVED** [review]

**Comment by kewitham** [comment]
> @kewitham Based on that doc, looks like this command will clean them up:
> 
> > If you want to manually run the command to add or update response templates without restarting the server, run this command:
> > ```shell
> > docker-compose run web python /code/crt_portal/manage.py update_response_templates
> > ```
> 
> which appears to be handled in our circleci workflow: https://github.com/usdoj-crt/crt-portal/blob/develop/.circleci/config.yml

@okkays that's the command I ran initially that only updates and creates the templates without deleting them.

**Comment by ilodidoj** [comment]
@kewitham I see this branch is created on your personal repository, probably not going to be a good idea to merge to usdoj portal repository.  I will validate the code, but you probably have to get latest branch from portal repository and then make changes there.  Let's talk before merging. CC: @okkays

**Comment by okkays** [comment]
> @kewitham I see this branch is created on your personal repository, probably not going to be a good idea to merge to usdoj portal repository. I will validate the code, but you probably have to get latest branch from portal repository and then make changes there. Let's talk before merging. CC: @okkays

When we click "Merge", we're actually "ignoring" her branch - github creates a new commit on `develop` that includes all of her changes. That's done as us (i.e. whoever clicks merge) using our permissions - so this will merge like any other branch (I've merged a couple of them for ADA to confirm).

It's also fairly straightforward for her to push a native branch, open a separate PR, and merge from that - so no big deal if we want to do that. Can discuss more next week 😄

**Comment by ilodidoj** [comment]
> s



> > @kewitham I see this branch is created on your personal repository, probably not going to be a good idea to merge to usdoj portal repository. I will validate the code, but you probably have to get latest branch from portal repository and then make changes there. Let's talk before merging. CC: @okkays
> 
> When we click "Merge", we're actually "ignoring" her branch - github creates a new commit on `develop` that includes all of her changes. That's done as us (i.e. whoever clicks merge) using our permissions - so this will merge like any other branch (I've merged a couple of them for ADA to confirm).
> 
> It's also fairly straightforward for her to push a native branch, open a separate PR, and merge from that - so no big deal if we want to do that. Can discuss more next week 😄

Let's discuss next week. thanks. :)

**Pull request closed without merging**